### DISCUSSION TITLE
希望作者多拉几天磨，快把离线翻译整出来

### DISCUSSION THREAD

**Discussion by 15169670031** [YiiGuxing/TranslationPlugin#5638] [General]
希望作者多拉几天磨，快把离线翻译整出来

**Comment by YiiGuxing** [comment]
真没多少时间啊，我最近在准备 v3.6 版本的工作，你看看我的提交时间，平均凌晨2、3点啊，最晚5点：
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/10737066/3f9548bd-2ddf-4cc2-bde9-c7216cedb14b)

白天还要上班，真挺不住啊，如果你真的需要离线翻译，可以用这个暂时先将就将就吧：#5411
